### ISSUE TITLE
Homebrew installation is broken

### ISSUE THREAD

**Issue by mondmann** [rhysd/Shiba#44]
$ brew cask install shiba

does not work for version 1.2.0 anymore, because of wrong sha256 checksum. Have you changed the binary after first release?

```
$ brew cask audit --download shiba
==> Downloading https://github.com/rhysd/Shiba/releases/download/v1.2.0/Shiba-darwin-x64.zip
######################################################################## 100.0%
==> Verifying checksum for Cask shiba
==> Note: running "brew update" may fix sha256 checksum errors
audit for shiba: failed
 - download not possible: Checksum for Cask 'shiba' does not match.

Expected: 8d5da51fba5ead6343e4f35532103c6c3cf28b649322d363b3ce188f62886bb4
Actual:   9df054565afadd0b7ae808dd8f47725bacb5737edb9db9ec635150a0398f603f
File:     /Users/xxx/Library/Caches/Homebrew/Cask/shiba--1.2.0.zip

To retry an incomplete download, remove the file above.
Error: audit failed for casks: shiba

```

**Comment by mondmann** [comment]
see also: https://github.com/Homebrew/homebrew-cask/pull/47544

**Comment by rhysd** [comment]
Thank you for reporting this issue and submitting fix to homebrew-cask. Yeah, installation was broken due to my package update without version bump. I'm sorry for inconvenience.

I released new version Shiba v1.2.1 with Electron version update. And I also submitted a PR to homebrew-cask https://github.com/Homebrew/homebrew-cask/pull/47812. It would solve this issue.
